app/Http/Controllers/PostController.php

```diff
@@ -50,31 +54,35 @@ public function store(PostRequest $request)
         }
 
         $post = Post::create($input);
-        return response(['post' => new
-        PostResource($post),
-            'message' => 'Success'], 200);
+
+        return response([
+            'post'    => new PostResource($post),
+            'message' => 'Success',
+        ], 200);
```

Comment:
Shouldn't we respond with [201](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201) here?